test/quicapitest.c

```diff
@@ -133,8 +133,13 @@ static int test_quic_write_read(int idx)
             goto end;
     }
 
-    if (!TEST_true(qtest_shutdown(qtserv, clientquic)))
-        goto end;
+    if (idx < 1)
+        /*
+         * Blocking SSL_shutdown cannot be tested here due to requirement to
+         * tick TSERVER during drainage.
```

Comment:
How do we test blocking SSL_shutdown then?

Reply:
Currently we can't. Actually this is a large blind spot in our testing right now &mdash; at some point I want the multistream test to be runnable in blocking mode. Probably will need to spin up a thread to tick the TSERVER and only support this test in builds with threads.